playbooks/hive-build-arm64/run.yaml

```diff
@@ -0,0 +1,14 @@
+- hosts: all
+  become: yes
+  roles:
+    - install-openjdk
+    - install-maven
+  tasks:
+    - name: install required packages
```

Comment:
This task name here should be "Build Hive" something.

Reply:
yes, my mistake, thanks for pointing out.